source/lib/ffgl/FFGL.h

```diff
@@ -104,7 +104,9 @@
 //include the appropriate OpenGL headers for the compiler
 
 #if defined( FFGL_WINDOWS )
-#define WIN32_LEAN_AND_MEAN//Exclude rarely-used stuff from Windows headers
+#ifndef WIN32_LEAN_AND_MEAN
+#      define WIN32_LEAN_AND_MEAN//Exclude rarely-used stuff from Windows headers
+#endif
 #define _WINSOCKAPI_       //Prevent inclusion of winsock
```

Comment:
We're not chaning here per-se but the `_WINSOCKAPI_`  is not used if `WIN32_LEAN_AND_MEAN` is used. So this is redundant.